### PR TITLE
[Footer] Update to mention that the site represents the undergraduate sub-committee

### PR DESCRIPTION
**Pull request by julianguyen**

### CODE REVIEW

File: themes/notmyidea/templates/base.html [hunk 1]
```diff
@@ -69,12 +69,12 @@
         {% endif %}
         </section><!-- /#extras -->
 
-        <footer id="contentinfo" class="body">
-                <address id="about" class="vcard body">
-                Proudly powered by <a href="http://getpelican.com/">Pelican</a>, which takes great advantage of <a href="http://python.org">Python</a>.
-                </address><!-- /#about -->
+         <footer id="contentinfo" class="body">
+            <address id="about" class="vcard body">
+                Women in Computer Science Undergraduate (WiCS UGrad) is a sub-committee of the University of Waterloo Women in Computer Science Committee. Check out their <a href="https://cs.uwaterloo.ca/wics">website</a>!
```
Comment (ehashman) [inline themes/notmyidea/templates/base.html]: Just tested this and it's looking pretty good! I'd change the blurb just slightly to read

"**The** Women in Computer Science Undergraduate **Committee** (WiCS **ugrad**) ..."

and WiCS proper is actually under the School of Computer Science, so I'd change "University of Waterloo" to "David R. Cheriton School of Computer Science" (or just Cheriton if you think that's too long).

**Comment by julianguyen** [comment]
How is this:
The Women in Computer Science Undergraduate (WiCS ugrad) is a sub-committee of the David R. Cheriton School of Computer Science Women in Computer Science Committee...etc

**Comment by julianguyen** [comment]
screen shot 2015-03-01 at 7 59 03 pm

**Comment by ehashman** [comment]
+11111111

**Comment by ehashman** [comment]
Great!

**Pull request merged**